core/src/message/context.rs

```diff
@@ -34,7 +34,9 @@ pub const OUTGOING_LIMIT: u32 = 1024;
 /// Context settings.
 #[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Decode, Encode, TypeInfo)]
 pub struct ContextSettings {
+    /// Amount of gas needed for send a message.
     sending_fee: u64,
+    /// Limit of outgoung messages that we can send before LimitExceeded.
```

Comment:
During one execution maybe?

Something like "Amount of messages can be produces from single message" isn't more explaining?

Reply:
You right, changed it.